.github/workflows/build-extensions.yml

```diff
@@ -102,12 +102,13 @@ jobs:
       - name: Build and test WASI-NN using ${{ matrix.compiler }} with ${{ matrix.build_type }} mode
         shell: bash
         run: |
-          source /opt/intel/openvino_2021/bin/setupvars.sh
           export Torch_DIR=$(pwd)/${PYTORCH_INSTALL_TO}/libtorch
+          export OpenVINO_DIR=./usr/lib/x86_64-linux-gnu/cmake/openvino2023.0.1
```

Comment:
Don't add `.` before `/usr/...`. It would be best to use an absolute path instead of a related one.

Reply:
I removed this line because it was not needed.